tools/serverpod_cli/lib/src/generator/dart/library_generators/model_library_generator.dart

```diff
@@ -160,15 +160,26 @@ class SerializableModelLibraryGenerator {
       }
 
       if (serverCode && tableName != null) {
-        classBuilder.extend =
-            refer('TableRow', 'package:serverpod/serverpod.dart');
+        classBuilder.implements.add(
+          refer('TableRow', 'package:serverpod/serverpod.dart'),
```

Comment:
Question: Don't we have a thing that builds references to the serverpod package? Instead of having this hardcoded?

I think it is `serverpodUrl(serverCode)`. Maybe we should change this to use that method as well.

Reply:
This will help us if we ever change the path to the Serverpod package.